src/test/librbd/test_mock_ObjectMap.cc

```diff
@@ -0,0 +1,272 @@
+// -*- mode:C; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
@dillaman super minor (not worth to repush, may be a separate commit someday to fix all the affected files):
It should be 'mode:C++' (causes some troubles to me when editing these files in Emacs).

Also, while we are here, 'indent-tabs-mode:t' makes my Emacs to use tabs, while I see the tabs are converted to spaces after your edit. I like spaces more too, though this statement overrides my emacs config...